community/ffmpeg/APKBUILD

```diff
@@ -50,7 +50,8 @@ build() {
 		--prefix=/usr \
 		--enable-avresample \
 		--enable-avfilter \
-		--enable-gnutls \
+		--enable-libtls \
+		--enable-nonfree \
```

Comment:
why is `--enable-nonfree` needed? can you mention this in ocmmit message?

Reply:
removed and fixed

Reply:
Added back because otherwise configure fails with `libtls is incompatible with the gpl and --enable-nonfree is not specified.`

Reply:
@ncopa I guess we could remove second commit now but libtls is in edge as well?